<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Activate</class>
 <widget class="QDialog" name="Activate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tell My - Activate</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>This is a trial version of Tell My. Get a key at tellmy.colten.me</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="continue_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue Trial</string>
   </property>
  </widget>
  <widget class="QPushButton" name="activate_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Activate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="key">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>XXXX-XXXX</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>If you have a key, enter it below.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="trial_info_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>You have n days left of your trial.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
